src/__experimental__/components/date-range/date-range.stories.js

```diff
@@ -21,6 +21,11 @@ export const Default = () => {
   const [state, setState] = useState(["2016-10-01", "2016-10-30"]);
   const startLabel = text("startLabel", "");
   const endLabel = text("endLabel", "");
+  const allowEmptyValueOnStartDate = boolean(
+    "allowEmptyValue on start date",
```

Comment:
can we rename this prop name to `allowEmptyValueOnStartDate `

Reply:
But it is not an actual prop, in this form its harder to read and it is confusing.
Here it clearly says what is happening and on which input.